### PR TITLE
feat(localization): Add localization

### PR DESCRIPTION
**Pull request by FranklinWaller**

### CODE REVIEW

File: src/js/localization.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const localization = {
+    COULD_NOT_DOWNLOAD_FRAGMENTS: {
+        nl: 'Kon video bestanden niet downloaden',
```
Comment (ivo-toby) [inline src/js/localization.js]: Shouldn't it be: "Kan videofragmenten niet downloaden" ?

Comment (ivo-toby) [reply src/js/localization.js]: poke!

**Comment by FranklinWaller** [comment]
Changed @buttonfreak

**Review by ivo-toby — APPROVED** [review]

**Comment by ivo-toby** [comment]
I'll rebase & publish

**Pull request merged**